frontend/src/modules/entitieslist/components/Entity.css

```diff
@@ -103,8 +103,8 @@
     color: #ffa10f;
 }
 
-.entity .sibling-entities {
-    color: #333941;
+.entity.sibling {
+    background-color: #313940;
```

Comment:
Seems like we don't use this color. Why now stick to `#333941` instead?

Reply:
I have no idea why I didn't use the previous color, changed it 👍🏾

Reply:
![image](https://user-images.githubusercontent.com/69736237/135731759-a0d7f8e5-2016-4c01-b596-1c4c7725ccee.png) 
I think I got the color from the mock because it is a little bit lighter than the default color let me know if this matters.

Reply:
Hmmmm... Right. Since the mockup was created, we changed the background color of the selected entity from lighter to darker than default.

I'll take the patch for a spin once it's done to see if there are further CSS changes needed.